### PR TITLE
Update UG DG

### PR DESCRIPTION
**Pull request by bangyiwu**
Updated the UG with some images of command behaviours, included annotations

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -248,12 +250,30 @@ Format: `sort -c INDEX`
 * Sorts through all contacts currently on screen and lists them according to the user-requested order
 * Index definitions:
 1. By lexicographical order of their names
-1. By lexicographical order of their address
-1. By lexicographical order of their email
+2. By lexicographical order of their address
+3. By lexicographical order of their email
```
Comment (fyshhh) [inline docs/UserGuide.md:254]: fyi markdown lists can have all indices be 1

File: docs/UserGuide.md [hunk 2]
```diff
@@ -342,10 +366,13 @@ Examples:
...
-![Edit Event Behaviour before](images/ug-images/edit-event-behaviour-before.png)
-After
-![Edit Event Behaviour after](images/ug-images/edit-event-behaviour-after.png)
+1. Entering edit command "edit -e 1 ap/1,2 rp/1,2" to edit the participants of CS2103 Meeting.
+
+![Edit Event Behaviour before](images/ug-images/eventBehaviourImages/edit-event-before-behaviour.png)
+
+2. Event participants successfully edited.
+
+![Edit Event Behaviour after](images/ug-images/eventBehaviourImages/edit-event-behaviour-after.png)
```
Comment (fyshhh) [inline docs/UserGuide.md:375]: naming is a bit weird... one is `before-behaviour` and the other `behaviour-after`

**Review by fyshhh — APPROVED** [review]
lgtm - i left some comments you could consider but honestly they don't affect anything

File: docs/team/bangyiwu.md [hunk 3]
```diff
@@ -82,7 +83,10 @@ Given below are my contributions to the project.
     
 
 * **Community**:
-  * PRs reviewed (with non-trivial review comments): [\#242](), [\#185](), [\#183]()
+  * PRs reviewed (with non-trivial review comments): e.g. [\#242](https://github.com/AY2021S1-CS2103T-W10-4/tp/pull/242), 
+  [\#185](https://github.com/AY2021S1-CS2103T-W10-4/tp/pull/185), 
+  [\#183](https://github.com/AY2021S1-CS2103T-W10-4/tp/pull/183)
   * Reported bugs and suggestions for other teams in the class (examples: https://github.com/bangyiwu/ped/issues)
+  * Reported bugs and suggestions for our own team member's implementations, e.g. PR [\#247](https://github.com/AY2021S1-CS2103T-W10-4/tp/pull/242) was created because of a bug I found
```
Comment (fyshhh) [inline docs/team/bangyiwu.md:90]: your pr says 247 but links to 242

Comment (bangyiwu) [reply docs/team/bangyiwu.md:90]: thanks for catching this!

**Pull request merged**